README.md

```diff
@@ -40,14 +42,16 @@ rm .install-filcrypto \
     ; FFI_BUILD_FROM_SOURCE=1 FFI_USE_BLST_PORTABLE=1 make
 ```
 
-By default, a 'gpu' option is used in the proofs library.  This feature is also used in FFI unless explicitly disabled.  To disable building with the 'gpu' dependency, set `FFI_USE_GPU=0`:
+By default, a 'gpu' option is used in the proofs library*.  This feature is also used in FFI unless explicitly disabled.  To disable building with the 'gpu' dependency, set `FFI_USE_GPU=0`:
 
 ```shell
 rm .install-filcrypto \
     ; make clean \
-    ; FFI_BUILD_FROM_SOURCE=1 FFI_USE_BLST=1 FFI_USE_GPU=0 make
+    ; FFI_BUILD_FROM_SOURCE=1 FFI_USE_PAIRING=1 FFI_USE_GPU=0 make
```

Comment:
Why using `FFI_USE_PAIRING=1`? Is it to demonstrate that this option exists?

Reply:
That's telling you how to enable pairing, now that it's not default.

Reply:
I see, it was in the wrong spot